.gitignore

```diff
@@ -54,4 +54,4 @@ docs/Gemfile.lock
 yarn.lock
 
 # Generated binaries
-**/bin
\ No newline at end of file
+**/_dist
```

Comment:
Done

Reply:
I `_dist` the more common approach as opposed to `bin`?